evap/results/templates/result_widget_tooltip.html

```diff
@@ -1,13 +1,29 @@
 {% load evaluation_filters %}
 
-{% if question_result.warning %}{% trans 'Only a few participants answered this question.' %}</br></br>{% endif %}
-{% with question_result.question.is_bipolar_likert_question as is_bipolar %}
-    {% for count, name, color, value in question_result.counts|zip_choices:question_result.choices %}
-        <span class='vote-text-{{ color }} fas fa-fw-absolute
-            {% if is_bipolar and value < 0 %}fa-caret-down pole-icon
-            {% elif is_bipolar and value > 0 %}fa-caret-up pole-icon
-            {% else %}fa-square{% endif %}'></span>
-        {{ name }}: {{ count }}/{{ question_result.count_sum }} ({{ count|percentage_one_decimal:question_result.count_sum }})
-        {% if not forloop.last %}</br>{% endif %}
-    {% endfor %}
-{% endwith %}
+{% if weight_info %}
+    <p>
+        {# QUESTION: Do we want to call a single result “evaluation” (currently) or “singel result” here? #}
+        {% blocktrans with percentage=weight_info %}
+          This evaluation contributes {{ percentage }} to the final grade of the course.
+        {% endblocktrans %}
```

Comment:
Do we want to call a single result “evaluation” (currently) or “single result” here?
I am in favor with calling everything consistently, but here I was thinking whether we should include more information here. On the index pages, I can go to the details page if I click on the result of a “full” evaluation – whereas single results are unclickable. May be confusing for some users or maybe I am overthinking here again.

Reply:
it can be called evaluation as it also is an evaluation and the information about being a single result is displayed in another place.